.gitmodules

```diff
@@ -4,9 +4,9 @@
 [submodule "dump978"]
 	path = dump978
 	url = https://github.com/ankurvdev/dump978
-[submodule "libusb"]
```

Comment:
The fork contains a few fixes for android (from https://github.com/libusb/libusb/pull/1164)  
I'd like to keep it as is until the PR is merged. 

Testing libusb for android is a little manual and time consuming task. Happy to do a merge with main if there's something missing in the fork